x-pack/test/security_solution_endpoint/page_objects/policy_page.ts

```diff
@@ -36,8 +36,7 @@ export function EndpointPolicyPageProvider({ getService, getPageObjects }: FtrPr
     async launchAndFindDeleteModal() {
       const actionsButton = await this.findFirstActionsButton();
       await actionsButton.click();
-      const deleteAction = await testSubjects.find('policyDeleteButton');
-      await deleteAction.click();
+      await testSubjects.click('policyDeleteButton');
```

Comment:
@dmlemeshko will `testSubjects.click()` wait for the test subject to appear before it attempts the click?

Reply:
Yes @charlie-pichette, it will. If you need to change default timeout, you can pass it this way:

`await testSubjects.click('policyDeleteButton', 25000);`